packages/@lwc/babel-plugin-component/src/component.js

```diff
@@ -68,8 +70,6 @@ module.exports = function ({ types: t }) {
     return {
         Program(path) {
```

Comment:
I left these in here because they are a component concern, but I'm not against consolidating them. Were you thinking `decorators/index.js` or the root `index.js`?

Reply:
I was thinking about keeping both here and just invoking the different functions in `index.js`.